readthedocs/oauth/migrations/0012_create_new_table_for_remote_organization_normalization.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.2.17 on 2020-12-23 10:58
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django_extensions.db.fields
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('socialaccount', '0003_extra_data_default_dict'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('oauth', '0011_add_default_branch'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='RemoteOrganization',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('pub_date', models.DateTimeField(auto_now_add=True, verbose_name='Publication date')),
+                ('modified_date', models.DateTimeField(auto_now=True, verbose_name='Modified date')),
```

Comment:
Is it possible to change these and use `TimeStampedModel` instead as you did with `RemoteRepositoryRelation`?

Same question for `RemoteRepository` :)